bigml/sourcehandler.py

```diff
@@ -86,6 +88,28 @@ def _create_remote_source(self, url, args=None):
         body = json.dumps(create_args)
         return self._create(self.source_url, body)
 
+    def _create_inline_source(self, src_obj, args=None):
+        """Create source from inline data
+
```

Comment:
The checking is right, but I miss some explanation in the method's doc saying what's the allowed structure and types for the src_obj arg.